### PR TITLE
[Mime] rename Part/BodyFile to Part/File

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Was proposed during a chat within the core-team.

/cc @javiereguiluz

### CODE REVIEW

**Review by maxbeckers — APPROVED** [review]

**Comment by maxbeckers** [comment]
Please reorder the use statements as the fabbot mentioned

**Review by chalasr — APPROVED** [review]
once fabbot is ok

**Comment by chalasr** [comment]
No markdown diff, did we forget to add some changelog entry?

**Comment by nicolas-grekas** [comment]
Updated!

File: src/Symfony/Component/Mime/Part/File.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class BodyFile
+class File
```
Comment (chalasr) [inline src/Symfony/Component/Mime/Part/File.php:19]: final btw?

Comment (nicolas-grekas) [reply src/Symfony/Component/Mime/Part/File.php:19]: https://github.com/symfony/symfony/pull/47462#discussion_r981268690

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**